README.rst

```diff
@@ -429,11 +429,27 @@ How to Cite
 
 If you utilize libCEED please cite::
 
-   @misc{libceed-dev-site,
-     title =  {lib{CEED} development site},
-     url =    {https://github.com/ceed/libceed},
-     howpublished = {\url{https://github.com/ceed/libceed}},
-     year = 2020
+   @misc{libceed-user-manual,
+     author       = {Abdelfattah, Ahmad and
+                     Barra, Valeria and
+                     Beams, Natalie and
+                     Brown, Jed and
+                     Camier, Jean-Sylvain and
+                     Dobrev, Veselin and
+                     Jean-Sylvain, Yohann and
```

Comment:
```suggestion
                     Dudouit, Yohann and
```

Reply:
Thanks! I fixed it here. Although the problem remains on [Zenodo])(https://zenodo.org/record/4302737/export/hx#.X8lEuVlKgUE) at the moment.